### PR TITLE
How to close an SSH session

### LINKED ISSUE #228: "Tried to do ssh to remote host but did not find my app folder at all." - from support
We should expand the docs to make it more clear how SSH sessions work.

My current answer was:

> The CI SSH box behaves the same way as normal CI jobs. You will need to execute checkout to checkout your repository.

### LINKED ISSUE #235: How to close an SSH session
Going to update https://docs.semaphoreci.com/article/75-debugging-with-ssh-access because of the release of `sem v0.9.0` https://github.com/semaphoreci/docs/issues/234.

### PR DESCRIPTION
**Pull request by mactsouk**
Closes #235 
Closes #228

### CODE REVIEW

File: debugging-with-ssh-access_5bbcbae0042863158cc73802.md [hunk 1]
```diff
@@ -112,14 +115,14 @@ The `http://localhost:6000` should now be accessible in your browser.
 
 ## Stopping a debug session
 
-To stop a debug session, run `sudo poweroff` or `sudo shutdown -r now` inside
-the Semaphore VM.
+To manually stop a debug session, you can run `sudo poweroff` or
+`sudo shutdown -r now` from the UNIX shell of the Semaphore VM. However, the
+debug session will *automatically* end once you exit the SSH session.
```
Comment (shiroyasha) [inline debugging-with-ssh-access_5bbcbae0042863158cc73802.md]: Let's change the order to follow the pattern of: "most common situation" down to "less common situations".

> The debug session will *automatically* end once you exit the SSH session.
>
> To manually stop a debug session you can run `sudo poweroff` or `sudo shutdown -r now` from 
> the shell of the Semaphore VM, or stop the debug session job from your local machine with 
> `sem stop job [job-id]`. You can find the ID of your debug job using `sem get jobs`.

Reasoning behind this is: We want to give the most usable information to the reader as soon as possible, with as little reading as possible.

File: debugging-with-ssh-access_5bbcbae0042863158cc73802.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ gives you the option to access all running jobs via SSH, to restart your jobs
 in debug mode, or to start on-demand virtual machines to explore the build
 environment.
 
-* [Setting Your SSH Key](#Setting Your SSH Key)
+* [Setting Your SSH Key](#setting-your-ssh-key)
```
[the remark this answers is not in the document]
Comment (mactsouk) [reply debugging-with-ssh-access_5bbcbae0042863158cc73802.md]: Done.

File: debugging-with-ssh-access_5bbcbae0042863158cc73802.md [hunk 3]
```diff
@@ -44,13 +44,17 @@ sem debug project [name-of-your-project]
...
+By default, the duration of the SSH session is limited to one hour. To run
+longer debug sessions, pass the `duration` flag to the above command:
 
 ``` bash
 sem debug project [name-of-your-project] --duration 3h
 ```
 
+By default, a debug session does not include the contents of the GitHub
+repository related to your Semaphore 2.0 project. However, you can manually
+get the contents of the GitHub repository by executing the `checkout` command.
```
[the remark this answers is not in the document]
Comment (mactsouk) [reply debugging-with-ssh-access_5bbcbae0042863158cc73802.md]: Done.

File: debugging-with-ssh-access_5bbcbae0042863158cc73802.md (hunk 1 above)
Comment (mactsouk) [reply debugging-with-ssh-access_5bbcbae0042863158cc73802.md]: Good point!

**Pull request merged**